packages/phone-number-privacy/common/src/utils/responses.utils.ts

```diff
@@ -0,0 +1,35 @@
+import Logger from 'bunyan'
```

Comment:
Having this in phone-number-privacy-common is a bit funky and may break the identity client (have not verified this) because it imports express, which will work if you are in a context where express is already installed like the signer or combiner, but not when for the ODIS client, which also imports phone-number-privacy-common. Even though it's duplicate code, I'd recommend having a version of this file in the signer and combiner packages respectively instead.

Reply:
You can also do `import type Logger from 'bunyan'` because if I read the file right you just need it for the interface, in which case I'm **fairly** sure the package is not actually imported at runtime. 

Alternatively, we could specify a minimal logger interface that's satisfied by `bunyan` in this context, and whatever logger gets passed from the ODIS client - if these code paths are used in that context as well.

Reply:
(To clarify, by "this" I mean "this file")